src/Microsoft.AspNetCore.Mvc.Razor/RazorViewEngineOptions.cs

```diff
@@ -31,7 +31,7 @@ public class RazorViewEngineOptions
         /// locate Razor files.
         /// </summary>
         /// <remarks>
-        /// At startup, this is initialized to include an instance of <see cref="PhysicalFileProvider"/> that is
+        /// At startup, this is initialized to include an instance of PhysicalFileProvider that is
```

Comment:
Ah, so the comment is really a bit wrong because it's going into an implementation detail of some other component. Anyway I think it's fine the way it is in this PR.